lib/src/components/Tab/tab.css.ts

```diff
@@ -0,0 +1,76 @@
+import { createTheme, style } from "@vanilla-extract/css";
+import { recipe } from "@vanilla-extract/recipes";
+import { TabTheme } from "./tab.types";
+
+export const [tabTheme, tabVars]: TabTheme = createTheme({
+  color: "#2F80ED",
+  background: "#BDDBFF60",
+  height: "50px",
+});
+
+export const tabHeaderContainerStyles = style({
+  height: tabVars.height,
+  position: "relative",
+  display: "flex",
+  alignItems: "center",
```

Comment:
Better to have used the `<Flex>` component gives us confidence when using our own components and we can find out potential flaws, keep this as least priority, can be ignored for this version of tabs